static/updatenotification.sh

```diff
@@ -19,32 +19,58 @@ debug_mode
 # Check if root
 root_check
 
-REPORTEDNCVERSION=""
+NCMIN=$(curl -s -m 900 $NCREPO/ | sed --silent 's/.*href="nextcloud-\([^"]\+\).zip.asc".*/\1/p' | sort --version-sort | grep "${CURRENTVERSION%%.*}" | tail -1)
+REPORTEDMAJ=""
+REPORTEDMIN=""
 
-if [ "$CURRENTVERSION" == "$NCVERSION" ]
+if [ "$CURRENTVERSION" == "$NCVERSION" ] && [ "$CURRENTVERSION" == "$NCMIN" ]
 then
     print_text_in_color "$IGreen" "You already run the latest version! ($NCVERSION)"
     exit
 fi
 
-if [ "$REPORTEDNCVERSION" == "$NCVERSION" ]
+if [ "$REPORTEDMAJ" == "$NCVERSION" ] && [ "$REPORTEDMIN" == "$NCMIN" ]
 then
-    print_text_in_color "$ICyan" "The notification regarding the new Nextcloud update has been already reported! ($NCVERSION)"
+    print_text_in_color "$ICyan" "The notification regarding the new Nextcloud update has been already reported!"
     exit
 fi
 
-if version_gt "$NCVERSION" "$CURRENTVERSION"
+if [ "$NCVERSION" == "$NCMIN" ] && version_gt "$NCMIN" "$REPORTEDMIN" && version_gt "$NCMIN" "$CURRENTVERSION"
 then
-    if crontab -l -u root | grep $SCRIPTS/update.sh
+    sed -i "s|^REPORTEDMAJ.*|REPORTEDMAJ=$NCVERSION|" $SCRIPTS/updatenotification.sh
+    sed -i "s|^REPORTEDMIN.*|REPORTEDMIN=$NCMIN|" $SCRIPTS/updatenotification.sh
```

Comment:
Why double notification here?

Reply:
This doesn't send a notification but substitutes both variables reportedmin and reportedmaj through new values to track which version got already reported and is defjnjtely needed here.

What sends the notification is notify_admin_gui